masterlist.yaml

```diff
@@ -3986,26 +3986,232 @@ plugins:
+          - '[Legacy of the Dragonborn SSE](https://www.nexusmods.com/skyrimspecialedition/mods/11802)'
+        condition: 'active("BUVARP.esp") and not active("DBM_BUVARP_Patch.esp")'
+      - <<: *hasPatchIncluded
+        subs:
+          - '[Cloaks of Skyrim](https://www.nexusmods.com/skyrimspecialedition/mods/6369/)'
+          - '[Legacy of the Dragonborn SSE](https://www.nexusmods.com/skyrimspecialedition/mods/11802)'
+        condition: 'active("Cloaks.esp") and not active("DBM_Cloaks_Patch.esp")'
       - <<: *hasPatchIncluded
         subs:
           - '[Cutting Room Floor](https://www.nexusmods.com/skyrimspecialedition/mods/276/)'
           - '[Legacy of the Dragonborn SSE](https://www.nexusmods.com/skyrimspecialedition/mods/11802)'
         condition: 'active("Cutting Room Floor.esp") and not active("DBM_CRF_Patch.esp")'
+      - <<: *hasPatchIncluded
+        subs:
+          - '[Daedric Cloaks](https://www.nexusmods.com/skyrimspecialedition/mods/9530/)'
+          - '[Legacy of the Dragonborn SSE](https://www.nexusmods.com/skyrimspecialedition/mods/11802)'
+        condition: 'active("DaedricCloaks.esp") and not active("DBM_DaedricCloaks_Patch.esp")'
+      - <<: *hasPatchIncluded
+        subs:
+          - '[Daedric Realms - Volume I The Hunting Grounds](https://www.nexusmods.com/skyrimspecialedition/mods/3140/)'
+          - '[Legacy of the Dragonborn SSE](https://www.nexusmods.com/skyrimspecialedition/mods/11802)'
+        condition: 'active("StagPrinceBow.esp") and not active("DBM_StagPrinceBow_Patch.esp")'
+      - <<: *hasPatchIncluded
+        subs:
+          - '[Dark Brotherhood for Good Guys](https://www.nexusmods.com/skyrimspecialedition/mods/519/)'
+          - '[Legacy of the Dragonborn SSE](https://www.nexusmods.com/skyrimspecialedition/mods/11802)'
+        condition: 'active("goodbrother.esp") and not active("DBM_GoodBrother_Patch.esp")'
+      - <<: *hasPatchIncluded
+        subs:
+          - '[Dawnguard and Clan Volkihar Epilogues](https://www.nexusmods.com/skyrimspecialedition/mods/12098/)'
+          - '[Legacy of the Dragonborn SSE](https://www.nexusmods.com/skyrimspecialedition/mods/11802)'
+        condition: 'active("Dawngaurd&VolkiharArtifactsQuests.esp") and not active("DBM_DGE_Patch.esp")'
+      - <<: *hasPatchIncluded
+        subs:
+          - '[Dawnguard and Clan Volkihar Epilogues](https://www.nexusmods.com/skyrimspecialedition/mods/12098/)'
+          - '[Legendary Skyrim Crossbows SSE](https://www.nexusmods.com/skyrimspecialedition/mods/8273/)'
+          - '[Legacy of the Dragonborn SSE](https://www.nexusmods.com/skyrimspecialedition/mods/11802)'
+        condition: 'active("Dawngaurd&VolkiharArtifactsQuests.esp.esp") and active("Legendary Skyrim Crossbows.esp") and not active("DBM_NAME_Patch.esp")'
+      - <<: *hasPatchIncluded
+        subs:
+          - '[Dynamic Dungeon Loot (DDL) - Random Loot Search](https://www.nexusmods.com/skyrimspecialedition/mods/10308/)'
```

Comment:
I'm not sure what effect (DDL) in the middle may have. Dynamic Dungeon Loot is fine.